source/help/passwords.md

```diff
@@ -5,28 +5,31 @@ arXiv uses two types of passwords:
 
 -   User account passwords for accessing your user account page
     ([https://arxiv.org/user](https://arxiv.org/user))
--   Paper passwords for claiming ownership of a paper submitted by
+-   Paper passwords for claiming ownership of an announced article submitted by
     co-authors or a proxy
 
 User account passwords:
 -----------------------
 
 -   Details about setting up your account are on our [registration help
-    page.](registerhelp.md)
+    page.](registerhelp)
 -   If you forgot your user account password, please see the [lost
     password form](https://arxiv.org/user/lost_password).
--   If you need help recovering an arXiv user account, please email
-    help@arxiv.org.
+-   If you need help recovering an arXiv user account, please [contact arXiv administrators](/help/contact)for support.
 
 Paper passwords:
 ----------------
 
-arXiv now maintains [authority records](authority.md). For any given
-paper, arXiv keeps a list of people who are authorized to make changes
-(these are referred to as **paper owners**). When a paper is first
-submitted to arXiv, the submitter is the one and only owner of the
-paper. See [authority records](https://arxiv.org/help/authority) for
-more information.
+arXiv maintains [authority records](authority) for the works within our corpus.
+These are the list of author accounts who are authorized to make changes (referred to as the **owner**).
```

Comment:
Maybe this should read, "These are the list of author accounts authorized to make changes (referred to as the **owner**)", removing "who are" since we're talking about accounts, not authors.